.gitignore

```diff
@@ -23,3 +23,5 @@ build/Release
 # Deployed apps should consider commenting this line out:
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules
+
+.idea/
```

Comment:
We shouldn't add IDE or even specific entries. You can put this into a global `.gitignore` on your system, see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore